packages/cloud/src/writeMatchStubHandler.ts

```diff
@@ -82,7 +82,6 @@ async function handler(file: any, _context: any) {
     stubs.forEach(async (stub) => {
       console.log(`processing stub ${stub.id}`);
       if (startTimeUTC) {
```

Comment:
i completely forgot what this was for.. is it safe to just remove the line below?

Reply:
Lol I've been trying to remember. I need to review what it's actually doing in the codebase now since I don't think it's still needed.